tests/foreman/api/test_reporttemplates.py

```diff
@@ -369,22 +380,76 @@ def test_negative_create_report_without_name():
 
 
 @pytest.mark.tier2
-@pytest.mark.stubbed
-def test_positive_applied_errata():
+@pytest.mark.rhel_ver_list([7, 8, 9])
```

Comment:
I do not like this approach. It will bite us in the ass once we start supporting RHEL 10. I suggest to change it the following way:
```suggestion
@pytest.mark.rhel_ver_match('^(?!6$)\d+$')
```